package.json

```diff
@@ -82,6 +82,8 @@
     "packages/**"
   ],
   "dependencies": {
+    "@dnd-kit/core": "^6.1.0",
```

Comment:
nämä voisi olla `packages/core/package.json` alla kun ei näitä kirjastoja varmaan muissa paketeissa tarvita

Reply:
hmm, ehkä rendering käyttää näitä kans? mä jätän varmuuden vuoks tähän.